src/Microsoft.Health.Fhir.Core/Utilities/ICallerIpAddressRetriever.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Utilities
+{
+    /// <summary>
+    /// Provides functionality to retrieve caller IP address.
+    /// </summary>
+    public interface ICallerIpAddressRetriever
```

Comment:
`Accessor` looks to be the AspNet suffix where the interface has only a property